Formula/r/rebar3.rb

```diff
@@ -12,18 +12,32 @@ class Rebar3 < Formula
   end
 
   bottle do
-    rebuild 1
-    sha256 cellar: :any_skip_relocation, arm64_sequoia: "5ceccb36cd664e34d69823f3ba77770da10c70218d69caafdb34ee3db3ce250b"
-    sha256 cellar: :any_skip_relocation, arm64_sonoma:  "68f8359445c350d021027633edf70f572f3000fee53b08142c0dd73169519072"
-    sha256 cellar: :any_skip_relocation, arm64_ventura: "f03cb9f3147876e10e2cd56dd8cf315b9f8b2188f808e8ad6018ab32839b0ea9"
-    sha256 cellar: :any_skip_relocation, sonoma:        "3c594f2b5a402ce44c8e55951d96e04e1762f2c3b054a6784da5172eb646e3fd"
-    sha256 cellar: :any_skip_relocation, ventura:       "7bc1cf24e4908fe6a41942ca8aaee98641d3b645a28fd31e9727e0ca0bfd9904"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "3049b6e29c6f38ac6ba4bbdfbdfd65e5a08e978b739957601c02aa6791cea5a5"
+    rebuild 2
+    sha256 cellar: :any_skip_relocation, arm64_sequoia: "6c00aba8ef8aa1954959241927a697dbb389af2e53113013ec740950859134b7"
+    sha256 cellar: :any_skip_relocation, arm64_sonoma:  "081a82ce85c737630c4f454e32031c095b5f7fb8c9139705c26ea1463b56bab2"
+    sha256 cellar: :any_skip_relocation, arm64_ventura: "daaf2e1860ada7bcaff3c59dd161611f4af4298893ee19d6b543523eecc060fd"
+    sha256 cellar: :any_skip_relocation, sonoma:        "c53cf3cf89d153a1c22c75077eb23e917806af58aac6ab58053537227a438970"
+    sha256 cellar: :any_skip_relocation, ventura:       "96c91363e3a00fc8b4cf1187601941bda4dff426e3e79f06f924225ea6de1c2f"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "33ffd13dc4476f1006a2b10b00fbdc818178f902299d0384a98362c929268954"
   end
 
+  depends_on "erlang@25" => [:build, :test]
```

Comment:
When Erlang/OTP 28 gets out will this be auto-bumped to 26?

Reply:
Or maybe the test below is enough to know something need to be updated. Yeah, once a year is not that bad (that the Erlang major cycle)

Reply:
> When Erlang/OTP 28 gets out will this be auto-bumped to 26?

Sadly no. We could make it something like
```ruby
depends_on "erlang@#{wanted_erlang_version}" => [:build, :test]
```
but then that doesn't work very well when we generate the package metadata that most `brew` users consume when they `brew install` things. (Most users won't have a copy of this repository checked out anymore.)

> Or maybe the test below is enough to know something need to be updated. Yeah, once a year is not that bad (that the Erlang major cycle)

Yes, that's the idea.

Reply:
I'll try to cook something up, if time allows, where I set `erlang` as a runtime dependency, which it is, but download the resources from GitHub directly (pre-compiled).

Reply:
> I'll try to cook something up, if time allows, where I set `erlang` as a runtime dependency, which it is, but download the resources from GitHub directly (pre-compiled).

Thanks for the offer, but we don't want to install pre-compiled stuff: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
Got it!